### ISSUE TITLE
Paragraph Feature Boxed Image alt text isn't being populated

### ISSUE THREAD

**Issue by nhjohnstone** [ATLAS-Illinois/Illinois-Framework-Theme#1319]
**Describe the bug**
Alt text isn't being added to the image because of the template code

**To Reproduce**
1. Go to [Feature Box](https://drupal.webtheme.illinois.edu/paragraphs/feature-box)
2. Make sure you are logged in
4. See error on any image from editora11y module

To Fix: on line 81 replace `img` tag with `{{ content.field_feature_boxed_img }}`

**Expected behavior**
There should be alt text on the image and therefore no error
